portal/src/components/FilterCategories/FilterCategories.component.js

```diff
@@ -115,34 +93,88 @@ export default {
         }
       )
     },
-    isShowCategoryItem({ category, item, indexItem }) {
-      return (
-        !item.is_empty && (category.showAll || indexItem < this.visible_items)
-      )
-    },
     getTitleTranslation(category, language) {
-      if (
-        !_.isNil(category.title_translations) &&
-        !_.isEmpty(category.title_translations)
-      ) {
+      if (category && category.title_translations) {
         return category.title_translations[language]
       }
       return category.name
+    },
+    datesRangeFilter() {
+      if (this.selectedFilters) {
+        const datesFilter = this.selectedFilters.find(
+          item => item.external_id === this.publisherDateExternalId
+        )
+        if (datesFilter && datesFilter.items) {
+          return {
+            start_date: datesFilter.items[0] || null,
+            end_date: datesFilter.items[1] || null
+          }
+        }
+      } else {
+        return {
+          start_date: null,
+          end_date: null
+        }
+      }
+    },
+    hasSelectedChildren(cat) {
+      return cat.children.filter(child => child.selected === true).length > 0
```

Comment:
`=== true` is hier overbodig

Reply:
En ik zou gebruiken maken van: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

Reply:
✅